lib/sequel/adapters/vertica.rb

```diff
@@ -32,7 +32,6 @@ def connect(server)
           :user => opts[:user],
           :password => opts[:password],
           :port => opts[:port],
-          :schema => opts[:schema],
```

Comment:
I believe the newest vertica gem doesn't allow specifying it anymore. I'll double-check.